kv/label.go

```diff
@@ -85,7 +92,7 @@ func (s *Service) findLabelByID(ctx context.Context, tx Tx, id influxdb.ID) (*in
 
 func filterLabelsFn(filter influxdb.LabelFilter) func(l *influxdb.Label) bool {
 	return func(label *influxdb.Label) bool {
-		return (filter.Name == "" || (filter.Name == label.Name)) &&
+		return (filter.Name == "" || (strings.EqualFold(filter.Name, label.Name))) &&
```

Comment:
is this change to maintain this with `utf8` chars? what's the motivation behind this change?

Reply:
yeah. our CI linter also seemed to agree with the change.

I had changed things originally to deal w/ finding labels case insensitively like so 
`strings.ToLower(filter.Name) == strings.ToLower(label.Name)` and CI called foul on that use.